### PR TITLE
added width - content fit

### PR DESCRIPTION
**Pull request by hkaur008**
## What kind of change does this PR introduce?

fix to #9718

## What is the current behavior?

image

## What is the new behavior?

image

### CODE REVIEW

File: apps/www/package.json [hunk 1]
```diff
@@ -16,22 +16,23 @@
...
     "common": "*",
+    "config": "*",
     "date-fns": "2.10.0",
     "dayjs": "^1.11.4",
     "globby": "^12.0.2",
     "gray-matter": "^4.0.2",
     "markdown-toc": "^1.2.0",
     "marked": "^4.0.12",
     "next": "latest",
+    "next-compose-plugins": "^2.2.1",
```
Comment (MildTomato) [inline apps/www/package.json:33]: @hkaur008 these changes look like they shouldn't be in here 🤔

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/www/package.json (hunk 1 above)
Comment (hkaur008) [reply apps/www/package.json:33]: Hey @MildTomato 
I also felt the same, the only useful section of the code which helped me to solve the issue was 
` html {
  scroll-behavior: smooth;
  width: fit-content;
}`

when i run npm build many of dev-dependencies were not there so they get added to package.json
however i can remove other changes .

and just do the significant code change ` html {
  scroll-behavior: smooth;
  width: fit-content;
}`

Comment (MildTomato) [reply apps/www/package.json:33]: did you run `npm i` in `/apps/www` or in the root `/`

We use turbo repo now, so npm installing should be done in the root, that might explain.
I'm not sure how this particular file would change though, these look like manual changes.

File: apps/www/styles/index.css [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 html {
   scroll-behavior: smooth;
+  width: fit-content;
```
Comment (MildTomato) [inline apps/www/styles/index.css:23]: hey @hkaur008 - could you remove all other changes apart from this? 😄

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/www/package.json (hunk 1 above)
Comment (MildTomato) [reply apps/www/package.json:33]: left request here https://github.com/supabase/supabase/pull/9903#discussion_r1007546363 😄

File: apps/www/styles/index.css (hunk 2 above)
Comment (hkaur008) [reply apps/www/styles/index.css:23]: sure !

**Pull request closed without merging**